src/base/hooks/use-license-file-status.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+
+import { useState } from 'react';
+
+import type { TableStatus } from '@/components/organisms/data-table/types';
+
+export const useTableStatus = (initialState?: TableStatus['get']): TableStatus => {
```

Comment:
Can we instead just use the `useTableFilters` hook. We will need to add a lot of filters in the future, and we do not envision to create a hook for each one of them.